tests/testLocalGalaxyTileObj.py

```diff
@@ -0,0 +1,463 @@
+import unittest
+import numpy as np
+import sqlite3
+import shutil
+import tempfile
+import os
+
+import lsst.utils.tests
+from lsst.utils import getPackageDir
+
+import lsst.sims.utils.htmModule as htm
+from lsst.sims.utils import xyz_from_ra_dec
+from lsst.sims.utils import ra_dec_from_xyz
+from lsst.sims.utils import angularSeparation
+from lsst.sims.utils import ObservationMetaData
+from lsst.sims.catUtils.baseCatalogModels import LocalGalaxyModels as LocGal
```

Comment:
I wasn't able to import this and couldn't find it. Should this be UWGalaxyModels?

Reply:
Yeah.  I changed the name of the classes before showing anyone else this code. Forgot to `grep` the unit tests.